src/screens/ChannelListScreen/elements/components/AvatarPostNotif.js

```diff
@@ -27,7 +27,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     flex: 1,
     borderWidth: 1,
-    borderColor: COLORS.white
+    borderColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The borderColor has been changed from `COLORS.white` to `COLORS.almostBlack`. Ensure that this change doesn't affect the visibility and contrast of the avatar border on different screens and themes (light/dark mode). If the application supports theme switching, consider using a dynamic color that adapts to the current theme.
```diff
-    borderColor: COLORS.white
+    borderColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->